nixpkgs-src.json

```diff
@@ -1,6 +1,6 @@
 {
     "owner":  "NixOS",
     "repo":   "nixpkgs",
-    "rev":    "ffea68a09cedab941d19f02c9041689ebc81192e",
-    "sha256": "1ipcgq111d11q78z9isfc00ndh7925asjkdjmb41ngfi9pabi0x3"
+    "rev":    "fda4b93cd4fd3775408117c380aab0f33737d30f",
```

Comment:
I'm fine with 18.03 if everyone else is fine with 18.03. Seems like this is a good enough reason to change branch.